### PR TITLE
Introduce DID verification

### PR DESCRIPTION
**Pull request by lemoustachiste**
This is an important one, 
as part of the v3 effort and in order to open the broader spectrum of SSI to blockcerts, this PR introduces support for DID verifications as used in the issuing and issuer's identity process.

The idea is fairly simple, we ensure that the issuing address is indeed in control of the issuer, programmatically as offered by the DID spec.

Things are still beta and this is a proof of concept that derived into a working solution.

There are items to improve (highlight the verification flow for instance, reference the url of the issuer profile to strengthen trust in the issuer's identity, etc), but as is, Blockcerts is capable of issuing (cert-issuer v3) with use of the verification method, and verifying the use of DIDs. We still would want to strengthen the trust in the issuer's identity by providing other mechanisms around the control of ownership of the keys.

This could include and is not restricted to:
- a challenge endpoint provided by the issuer in the did document that CVJS interrogates
- a merkle root of known issuing addresses (past and present). We either trust the root issuer (CA) of this merkle root or enhance that with DPKI solutions like key chains for instance.
- smart contract as a decentralized CA
- well known standard on the issuer's domain
- ...

### CODE REVIEW

**Review by raiseandfall — COMMENTED** [review]
Looks pretty good, just one feedback.

File: src/constants/verificationSubSteps.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  checkReceipt = 'checkReceipt',
+  checkIssuerSignature = 'checkIssuerSignature',
+  checkIssuerIdentity = 'checkIssuerIdentity',
+  checkAuthenticity = 'checkAuthenticity',
+  checkRevokedStatus = 'checkRevokedStatus',
+  checkExpiresDate = 'checkExpiresDate'
+}
+
+function getTextFor (subStep: string, status: string): string {
+  return i18n['en-US'].subSteps[`${subStep}${status}`];
```
[the remark this answers is not in the document]
Comment (lemoustachiste) [reply src/constants/verificationSubSteps.ts:32]: Yeah we are getting the text for the substep corresponding to its status. I think that's always been done like this. We are just getting the default text I assume to pregenerate the architecture of the steps. And later on there will be another translation? I am not sure it's a bit fuzzy.

[the remark this answers is not in the document]
Comment (raiseandfall) [reply src/constants/verificationSubSteps.ts:32]: ah right, that sounds familiar. It's been a while.

**Review by raiseandfall — APPROVED** [review]

**Pull request merged**